recipes-kernel/linux/files/bcm2712-raspberrypi5-xen.dtso

```diff
@@ -60,3 +60,5 @@
 &bcm_reset      { xen,passthrough; };
 &pcie_rescal    { xen,passthrough; };
 &mip0           { xen,passthrough; };
+&rp1_usb0       { xen,passthrough; };
+&rp1_usb1       { xen,passthrough; };
```

Comment:
There is an issue - if MMC is used by Dom0 it should not have xen,passthrough; property, otherwise Dom0 will crash with XEN IO exception.

Reply:
fixed